### PR TITLE
Fixes #100 sysdig-cloud ADDITIONAL_CONF support

### LINKED ISSUE #100: sysdig-cloud ADDITIONAL_CONF support
It would be great if the sysdig-cloud item supported the ADDITIONAL_CONF, so we could disable collection of services we don't need.

http://support.sysdigcloud.com/hc/en-us/articles/206443225-

### PR DESCRIPTION
**Pull request by jwhitcraft**
- Support the ADDITIONAL_CONF flag env variable for the sysdig container.

### CODE REVIEW

File: templates/sysdig-cloud/0/rancher-compose.yml [hunk 1]
```diff
@@ -25,3 +25,8 @@
       type: "string"
       default: sysdig.exclude_sysdigcloud=true
       required: true
+    - variable: "SDC_ADDITIONAL_CONF"
+      label: "Additional Configuration"
+      description: "Content must be valid yaml code, use the escape \n parameter for new lines and adhere to proper indent requirements."
+      type: "string"
```
Comment (cloudnautique) [inline templates/sysdig-cloud/0/rancher-compose.yml:31]: Would the (multi-line)[http://docs.rancher.com/rancher/catalog/] type work better? I think we add new lines etc.

Comment (jwhitcraft) [inline templates/sysdig-cloud/0/rancher-compose.yml:31]: According to their docs you have to have returns be \n, so I'm not sure if multi line would work. I can create a test later today.

Comment (cloudnautique) [inline templates/sysdig-cloud/0/rancher-compose.yml:31]: Any update?

Comment (jwhitcraft) [inline templates/sysdig-cloud/0/rancher-compose.yml:31]: Sorry, I did test this and it does need the new lines as \n and not the ascii-13 char.

**Pull request merged**